src/components/node-list/node-list-row.js

```diff
@@ -88,7 +88,8 @@ const NodeListRow = ({
           `pipeline-row__toggle--kind-${kind}`,
           {
             'pipeline-row__toggle--disabled': disabled,
-            'pipeline-row__toggle--selected': selected
+            'pipeline-row__toggle--selected': selected,
+            'pipeline-row__toggle--cursor': type !== 'tag'
```

Comment:
Not sure if we should use `cursor: pointer` on tags, as I think it would need to be all or nothing. Using it for all would then mean the whole row, which I don't think will feel right. If it's about consistency with the eye button behaviour, I'd probably remove `cursor: pointer` from them too (but don't think that change is really needed).

Reply:
thanks for raising this point! 

We actually discussed about this in our earlier discussion between Gabriel, Richard and I, and we decided to keep the 'hand' cursor (the pointer cursor) for the eye icon given that that icon is in fact an actual toggle button that the user can click/toggle, and the hand cursor within our app will be an indication for that type of 'toggle' button. ( This is also one of the thinking behind this ticket, which is to extract this type of interactive toggle elements out between the filters and nodes on the side bar).

Having played around with both scenerio when I was testing with the sidebar, I also personally think having the pointer cursor for the eye icon to indicate to the user that they can  toggle the button is helpful too :) but again, happy to discuss otherwise :D

Reply:
The cursor on the eye is fine, I meant on tags but it's a minor detail so not a big deal.

Reply:
I don't think the `cursor:pointer` is applied on tags, hence this new class to check and only apply to non-tag rows?